spec/turn_spec.rb

```diff
@@ -0,0 +1,139 @@
+require 'rspec'
+require './lib/card'
+require './lib/deck'
+require './lib/player'
+require './lib/turn'
+
+RSpec.describe Turn do
```

Comment:
It's great that you used helper methods for `war_winner` and `basic_winner`. Those methods should also be explicitly tested in our spec file as well.